domains.txt

```diff
@@ -140,9 +160,11 @@ connectusnow.com
 consolpartners.com
 consortia.co.uk
 consultia.eu.com
+consultsolutions.co.uk
```

Comment:
Consult Solutions is a UK recruitment consultancy who specialise in the placement of Information Technology, Energy and Senior Appointment professionals.